tests/RobotFramework/config_management/conf_child_mgmt.robot

```diff
@@ -0,0 +1,186 @@
+*** Settings ***
+# Library    Dialogs
+Library    SSHLibrary
+Library    DateTime
+Library    MQTTLibrary
+Library    CryptoLibrary    variable_decryption=True
+Library    RequestsLibrary
+Library    REST    https://qaenvironment.eu-latest.cumulocity.com
+Library    JSONLibrary
+Library    Collections
+
+*** Variables ***
+
+${PARENT_IP}             192.168.1.120
+${CHILD_IP}              192.168.1.200
+${HTTP_PORT}             8000
+${USERNAME}              pi
+${PASSWORD}              crypt:LO3wCxZPltyviM8gEyBkRylToqtWm+hvq9mMVEPxtn0BXB65v/5wxUu7EqicpOgGhgNZVgFjY0o=
+${url_tedge}             qaenvironment.eu-latest.cumulocity.com
+${user}                  systest_preparation
```

Comment:
Just out of curiosity: is there any semantics difference in the variable names in uppercase like `USERNAME` and lowercase like `user` here? Are the uppercase ones used an environmental variables or something like that?